.github/workflows/ci.yml

```
@@ -380,20 +379,20 @@ jobs:
                 shell: bash
                 run: |
                     set -o pipefail
-                    dotnet test tests/squirix.server/squirix.server.smoke-tests/Squirix.Server.SmokeTests.csproj -f net10.0 --configuration Release --no-build --verbosity normal \
-                      --blame-hang-timeout 5m \
-                      --filter "FullyQualifiedName~AuthSmokeTests" 2>&1 | tee test-output-smoke-auth.log
+                    dotnet test --project tests/squirix.server/squirix.server.smoke-tests/Squirix.Server.SmokeTests.csproj --framework net10.0 --configuration Release --no-build --verbosity normal \
+                      --long-running 300 \
+                      --filter-class AuthSmokeTests 2>&1 | tee test-output-smoke-auth.log
 
             -   name: Run Smoke Tests
```

YAMLlint (1.37.1)

[error] too many spaces after hyphen

(hyphens)